tools/restore/restore.sh

```diff
@@ -220,10 +231,37 @@ function restore_database {
     echo "RESTORE OF $db COMPLETE"
 }
 
+function activate_gcp() {
+  echo "Activating google credentials before beginning"
+  gcloud auth activate-service-account --key-file "/credentials/gcp/credentials.json"
```

Comment:
See above comment.   Key file can be in a single stable mount path that's predictable, irrespective of cloud.   I.e. `/credentials/secret`